src/presence_analyzer/static/js/presence_weekday.js

```diff
@@ -0,0 +1,31 @@
+google.load("visualization", "1", {packages:["corechart"], 'language': 'en'});
+
+(function($) {
+    $(document).ready(function(){
+        var loading = $('#loading');
+        $.getJSON("/api/v1/users", function(result) {
+            var dropdown = $("#user_id");
+            $.each(result, function(item) {
+                dropdown.append($("<option />").val(this.user_id).text(this.name));
+            });
+            dropdown.show();
+            loading.hide();
+        });
+        $('#user_id').change(function(){
+            var selected_user = $("#user_id").val();
+            var chart_div = $('#chart_div');
+            if(selected_user) {
+                loading.show();
+                chart_div.hide();
+                $.getJSON("/api/v1/presence_weekday/"+selected_user, function(result) {
+                    var data = google.visualization.arrayToDataTable(result);
+                    var options = {};
+                    chart_div.show();
+                    loading.hide();
+                    var chart = new google.visualization.PieChart(chart_div[0]);
+                    chart.draw(data, options);
+                });
+            }
+        });
+    });
+})(jQuery);
```

Comment:
This shouldn't be here, configure ur editor
![image](https://cloud.githubusercontent.com/assets/5078402/25038227/db58daa8-20fd-11e7-94c6-c6bec639be61.png)